frontend/src/lib/i18n/locales/en.json

```diff
@@ -0,0 +1,61 @@
+{
+	"appbar": {
+		"app_name": "Language Depot"
+	},
+	"appmenu": {
+		"log_out": "Log out"
+	},
+	"login": {
+		"missing_user_info": "User info missing",
+		"password_missing": "Password missing",
+		"page_header": "Log in",
+		"label_email": "Email (or Send/Receive username)",
+		"label_password": "Password",
+		"bad_credentials": "Something went wrong, please make sure you have used the correct account information.",
+		"button_login": "Log in",
+		"button_login_again": "Try to log in again?"
+	},
+	"user_dashboard": {
+		"page_header": "My projects"
+	},
+	"projectlist": {
+		"shared_with": "Shared with {memeberCount, plural, one {no one} other {# people} }",
```

Comment:
I presume this did not work unless it's just positionally based since there's a typo in `memeberCount`.  Impressive that it appears the lib is utilizing intl's plural rules under the covers 🤩

Reply:
yeah it's probably not going to work properly because of that. But yes it does use the proper plural rules based on ICU.